swdb_2019_tools/EM_mesh_extract_crummy.ipynb

```diff
@@ -0,0 +1,63 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "'''Mesh extraction function for EM connectivity data'''"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def generate_mesh_list(cells_df):\n",
```

Comment:
you should take this function and copy it into a seperate .py file, rather than a .ipynb file so that other people can import it.

Reply:
Maybe later?